video/out/wayland_common.c

```diff
@@ -2234,24 +2234,6 @@ void vo_wayland_set_opaque_region(struct vo_wayland_state *wl, int alpha)
     }
 }
 
-bool vo_wayland_supported_format(struct vo *vo, uint32_t drm_format, uint64_t modifier)
```

Comment:
You need to remove this from `wayland_common.h` as well.

Reply:
done